### PR TITLE
change reboot cmds

### PR DESCRIPTION
**Pull request by abhinavdahiya**
#14 

/cc: @xiang90 @crawford

### CODE REVIEW

File: chaos/cluster/cluster.go [hunk 1]
```diff
@@ -21,11 +21,24 @@ import (
...
+DefaultDependencies=no
+Before=sysinit.target
 
-	cmdSystemUp = "sudo systemctl is-active kubelet"
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/sleep %ds
+
+[Install]
+WantedBy=multi-user.target
```
Comment (crawford) [inline chaos/cluster/cluster.go]: This should be `sysinit.target`.

File: chaos/cluster/cluster.go [hunk 2]
```diff
@@ -21,11 +21,24 @@ import (
...
+`
+	cmdEnableStallService  = "sudo mv /tmp/stall.service /etc/systemd/system/stall.service && sudo systemctl daemon-reload && sudo systemctl enable stall.service"
+	cmdDisableStallService = "sudo systemctl disable stall.service && sudo mv /etc/systemd/system/stall.service /tmp/stall.service && sudo systemctl daemon-reload"
```
Comment (crawford) [inline chaos/cluster/cluster.go]: It's fine to leave the service in its location under `/etc`. `systemctl disable` will be sufficient.

**Review by crawford — APPROVED** [review]
A few NITs, but it looks good overall.

File: chaos/cluster/cluster.go (hunk 1 above)
Comment (abhinavdahiya) [reply chaos/cluster/cluster.go]: done

File: chaos/cluster/cluster.go (hunk 2 above)
Comment (abhinavdahiya) [reply chaos/cluster/cluster.go]: done

**Comment by xiang90** [comment]
lgtm

**Review by crawford — APPROVED** [review]

**Pull request merged**